<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IView</class>
 <widget class="QMainWindow" name="IView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>711</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>iView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="graphicsLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoadImageFromDrive"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDragMode"/>
   <addaction name="actionSkyMode"/>
   <addaction name="actionWCSMode"/>
   <addaction name="separator"/>
   <addaction name="actionImage_statistics"/>
   <addaction name="actionFinder"/>
   <addaction name="actionRedshift"/>
   <addaction name="separator"/>
   <addaction name="actionSourceCat"/>
   <addaction name="actionRefCat"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionStart"/>
   <addaction name="actionEnd"/>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
  </widget>
  <action name="actionLoadImageFromDrive">
   <property name="text">
    <string>&amp;Load image from drive</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_start.png</normaloff>:/icons/icon_start.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>First in series</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_previous.png</normaloff>:/icons/icon_previous.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Previous in series</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_back.png</normaloff>:/icons/icon_back.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Play back series</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_forward.png</normaloff>:/icons/icon_forward.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Play forward series</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_next.png</normaloff>:/icons/icon_next.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Next in series</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icon_end.png</normaloff>:/icons/icon_end.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Last in series</string>
   </property>
  </action>
  <action name="actionSourceCat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SourceCat</string>
   </property>
   <property name="toolTip">
    <string>Overlay source catalog</string>
   </property>
  </action>
  <action name="actionRefCat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RefCat</string>
   </property>
   <property name="toolTip">
    <string>Overlay reference catalog</string>
   </property>
  </action>
  <action name="actionDragMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/hand.png</normaloff>:/icons/hand.png</iconset>
   </property>
   <property name="text">
    <string>Drag Mode</string>
   </property>
   <property name="toolTip">
    <string>Middle mouse button drags image</string>
   </property>
  </action>
  <action name="actionSkyMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/pipette.png</normaloff>:/icons/pipette.png</iconset>
   </property>
   <property name="text">
    <string>Sky Mode</string>
   </property>
   <property name="toolTip">
    <string>Middle mouse button creates sky measurement areas</string>
   </property>
  </action>
  <action name="actionWCSMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/wcs.png</normaloff>:/icons/wcs.png</iconset>
   </property>
   <property name="text">
    <string>WCS Mode</string>
   </property>
   <property name="toolTip">
    <string>Middle mouse button changes CRPIX1/2</string>
   </property>
  </action>
  <action name="actionImage_statistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/sigma.png</normaloff>:/icons/sigma.png</iconset>
   </property>
   <property name="text">
    <string>Image statistics</string>
   </property>
   <property name="toolTip">
    <string>Display local and global image statistics</string>
   </property>
  </action>
  <action name="actionMaskingMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/satellite-icon.png</normaloff>:/icons/satellite-icon.png</iconset>
   </property>
   <property name="text">
    <string>Masking</string>
   </property>
   <property name="toolTip">
    <string>Mask satellites and other image defects</string>
   </property>
  </action>
  <action name="actionFinder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/magnifyer.png</normaloff>:/icons/magnifyer.png</iconset>
   </property>
   <property name="text">
    <string>Finder</string>
   </property>
   <property name="toolTip">
    <string>Locate a target in the image</string>
   </property>
  </action>
  <action name="actionRedshift">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/spec.png</normaloff>:/icons/spec.png</iconset>
   </property>
   <property name="text">
    <string>Redshift</string>
   </property>
   <property name="toolTip">
    <string>Redshift estimation for FITS spectra</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
